ui/pages/onboarding-flow/import-srp/import-srp.js

```diff
@@ -56,11 +56,11 @@ export default function ImportSRP({ submitSecretRecoveryPhrase }) {
             <a
               key="learnMore"
               type="link"
-              href="https://metamask.zendesk.com/hc/en-us/articles/360036464651"
+              href="https://metamask.zendesk.com/hc/en-us/articles/360060826432-What-is-a-Secret-Recovery-Phrase-and-how-to-keep-your-crypto-wallet-secure"
```

Comment:
[REQ] Sorry for this, i know its a nit but it is pretty helpful...could we move this to the zendesk urls constant in the constants folder?

Reply:
@brad-decker Fixed in: aad65a42b3056b558751a14d92e6edc5c7aceeba